### DISCUSSION TITLE
Test Discussion

### DISCUSSION THREAD

**Discussion by KellenWatt** [FRC-Team-2165/2022#2] [General]
This is a test of discussions to check if there are notifications.

**Comment by ericwetzel7** [comment]
I didnt get a notification, but I'm still checking the discussion every once and a while so i'll see the messages

**Reply by KellenWatt** [reply]
Good to know! Honestly, I wrote this just to see exactly how this worked, and I meant to delete it. 

Regardless, you seem to have answered that question. There probably aren't notifications.

**Reply by ericwetzel7** [reply]
when you commented back I got an email notification on my phone, but I don't see on in my inbox for the original post, so there might be some form of notifications
